src/Chapter2.hs

```diff
@@ -501,7 +518,9 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 :: (Eq a, Num a) => [a] -> Bool
+isThird42 [] = False
+isThird42 ( _ : _ : x : xs ) = if x == 42 then True else False
```

Comment:
You don't need to compare `x` with 42 separately, you can pattern match on `42` directly in the pattern.

```suggestion
isThird42 ( _ : _ : 42 : _ ) = True
isThird42 _ = False
```